README.md

```diff
@@ -32,6 +32,10 @@ which yarn
 
 If any installation step failed you'll see `______ not found`. If you don't see that then you're ready to continue on your quest, fellow coder.
 
+The next step would be to add express to your project, which can be done using a simple yarn command `sh yarn add express `
```

Comment:
Is the `sh` really needed? I don't imagine most Windows or Mac users will have access to it.